engine/settings/base.py

```diff
@@ -92,11 +92,11 @@ class DatabaseTypes:
     },
 }
 
-DATABASE_NAME = os.getenv("DATABASE_NAME") or os.getenv("MYSQL_DB_NAME")
-DATABASE_USER = os.getenv("DATABASE_USER") or os.getenv("MYSQL_USER")
-DATABASE_PASSWORD = os.getenv("DATABASE_PASSWORD") or os.getenv("MYSQL_PASSWORD")
-DATABASE_HOST = os.getenv("DATABASE_HOST") or os.getenv("MYSQL_HOST")
```

Comment:
the `MYSQL_*` env vars are not referenced anywhere in the codebase. Lets simplify things by just using the `DATABASE_*` env vars.

Reply:
Let's keep `MYSQL_*` vars for now, because this is needed for backwards compatibility. 
I should've add some comments here when I was working on this, my bad.